lib/linked_in/organization.rb

```diff
@@ -9,10 +9,21 @@ def organization(options = {})
       get(path, options)
     end
 
+    def organization_profile(options = {})
```

Comment:
```suggestion
    def organizations_lookup(options = {})
```

IMO we should keep the function name similar to the endpoint name. Here the result is an array and the function name looks like it is an object.